### DISCUSSION TITLE
I want to not to select any candidates unless I type ‘Enter’.

### DISCUSSION THREAD

**Discussion by moyotar** [company-mode/company-mode#1183] [Q&A] [answered]
![image](https://cloud.githubusercontent.com/assets/22092607/19632512/ca1c3920-99d9-11e6-995a-b9e5e44c1439.png)
Like pic above, It will select the first candidate automatically if I type `SPACE`. I don't want this, because what I really want is just insert `SPACE` and skip the completion.

**Comment by dgutov** [comment, marked as answer]
Whether `SPACE` inserts the first candidate, depends on the value of `company-auto-complete`, and on whether you've interacted with the current popup.

**Comment by moyotar** [comment]
Thanks!
